test/run

```diff
@@ -10,10 +10,28 @@ test/common/pixel-tests pull
 
 PREPARE_OPTS=""
 RUN_OPTS=""
+ALL_TESTS="$(test/common/run-tests --test-dir test/verify -l)"
 
 # every known case needs to set RUN_OPTS to something non-empty, so that we can check if we hit any branch
 case "${TEST_SCENARIO:-}" in
-    "") RUN_OPTS=" " ;;&
+    "")
```

Comment:
This won't match on (for example) "pybridge".

Reply:
That might have been the idea, no?  "fedora-38/pybridge" runs everything, and we never use "fedora-38/pybridge-networking".  Maybe `""|"*default*")` so that we can use `pybridge-default`?

Reply:
Yes, that was the idea. This is only for the "empty test scenario" case, but I think you found a better way in #18950. The intention is absolutely that you can combine scenarios, like "pybridge-default" or "firefox-networking". I like the idea of letting "" run everything as before, and explicitly introduce a "/default" scenario.

Reply:
Fixed.